transformation_test/testdata/logging_processor-tomcat-system/output_fluentbit.yaml

```diff
@@ -1,136 +1,224 @@
+      module: main
+      source: org.apache.catalina.startup.HostConfig
     labels:
       logging.googleapis.com/instrumentation_source: agent.googleapis.com/tomcat_system
     logName: projects/my-project/logs/transformation_test
-    timestamp: now
+    severity: 200.0
+    timestamp: 2025-07-21T01:02:43.555000000Z
   - jsonPayload:
-      message: "Jul 21 14:40:01 myhost systemd[1]: logrotate.service: Succeeded."
+      level: INFO
+      message: org.apache.coyote.AbstractProtocol Starting ProtocolHandler ["http-nio-8080"]
+      module: main
+      source: org.apache.coyote.AbstractProtocol
     labels:
       logging.googleapis.com/instrumentation_source: agent.googleapis.com/tomcat_system
     logName: projects/my-project/logs/transformation_test
-    timestamp: now
+    severity: 200.0
+    timestamp: 2025-07-21T01:02:43.565000000Z
   - jsonPayload:
-      message: "Jul 21 14:40:01 myhost systemd[1]: Finished Rotate log files."
+      level: INFO
+      message: org.apache.catalina.startup.Catalina Server startup in 356 ms
+      module: main
+      source: org.apache.catalina.startup.Catalina
     labels:
       logging.googleapis.com/instrumentation_source: agent.googleapis.com/tomcat_system
     logName: projects/my-project/logs/transformation_test
-    timestamp: now
+    severity: 200.0
+    timestamp: 2025-07-21T01:02:43.565000000Z
   - jsonPayload:
-      message: "Jul 21 16:10:35 myhost sudo:     bob : TTY=pts/0 ; PWD=/home/bob ; USER=root ; COMMAND=/bin/systemctl restart apache2"
+      level: WARN
+      message: org.apache.catalina.authenticator.AuthenticatorBase Security checking request failed for /admin/
+      module: http-nio-8080-exec-2
+      source: org.apache.catalina.authenticator.AuthenticatorBase
     labels:
       logging.googleapis.com/instrumentation_source: agent.googleapis.com/tomcat_system
     logName: projects/my-project/logs/transformation_test
-    timestamp: now
+    timestamp: 2025-07-21T10:33:41.123000000Z
   - jsonPayload:
-      message: "Jul 21 16:10:35 myhost sudo: pam_unix(sudo:session): session opened for user root by bob(uid=0)"
+      level: ERROR
+      message: org.apache.catalina.core.StandardWrapperValve Servlet.service() for servlet [dispatcher] threw exception
+      module: http-nio-8080-exec-4
+      source: org.apache.catalina.core.StandardWrapperValve
     labels:
       logging.googleapis.com/instrumentation_source: agent.googleapis.com/tomcat_system
     logName: projects/my-project/logs/transformation_test
-    timestamp: now
+    timestamp: 2025-07-21T10:33:44.234000000Z
   - jsonPayload:
-      message: "Jul 21 16:12:01 myhost sudo: pam_unix(sudo:session): session closed for user root"
+      message: 21-Jul-2025 14:40:01.456 INFO [ContainerBackgroundProcessor[StandardEngine[Catalina]]] org.apache.catalina.core.StandardContext Reloading Context [/myapp] has started
```

Comment:
Nit : I'm realizing that `[ContainerBackgroundProcessor[StandardEngine[Catalina]]]` won't be parsed correctly by `\[(?<module>[^\]]+)\]` due to the nested brackets.

https://github.com/GoogleCloudPlatform/ops-agent/blob/cda38309450bf45aeffc5706fcf60adf179604ff/apps/tomcat.go#L79-L83

**I don't think we should update/address/fix in this PR. This is more of a curiosity about `Apache Tomcat` versions. Does the application version of matter when getting this type of "nested bracket" logs ?**

Reply:
Will leave this as is for now

Reply:
I agree! We should leave it as is for now. I was mostly curious if this was due to the logs from a different Tomcat version.